RTNeural/dense/dense_eigen.h

```diff
@@ -141,7 +149,14 @@ class DenseT
     /** Performs forward propagation for this layer. */
     inline void forward(const Eigen::Matrix<T, in_size, 1>& ins) noexcept
     {
-        outs.noalias() = weights * ins + bias;
+        for (int i = 0; i < in_size; ++i)
+            ins_internal(i, 0) = ins(i, 0);
```

Comment:
It would be great if there were a way to do this kind of "one-multiply" dense layer operation without requiring the copy here... Unfortunately I can't think of an easy way at the moment.

Maybe it would be possible to set up something at a higher level which makes sure that all the layer input/output matrices have enough memory available to be extended to be one row longer? That way we could do something like an `Eigen::Map` to "grow" the input vector by one row, and just put a `1` in that row, which would avoid the copy. That seems a bit cumbersome though, so it would be cool if there were an easier way.